frontend/rpm/template.go

```diff
@@ -140,18 +140,18 @@ OrderWithRequires(postun): systemd
 func (w *specWrapper) Requires() fmt.Stringer {
 	b := &strings.Builder{}
 
+	artifacts := w.GetArtifacts(w.Target)
+	if artifacts == nil || artifacts.IsEmpty() {
+		return b
+	}
```

Comment:
I don't think we can return early here since this is where dependencies get written.

```suggestion
```

Reply:
good catch, this was not supposed to be added here